Switcher/TargetInformationBuilder.php

```diff
@@ -50,16 +50,18 @@ public function __construct(Request $request, RouterInterface $router, $allowedL
      * Builds a bunch of informations in order to build a switcher template
      * for custom needs
      *
-     * Will return something like this (let's say current locale is fr :
+     * Will return something like this (let's say current locale is fr_FR :
      *
      * current_route: hello_route
-     * current_locale: fr
+     * current_locale: fr_FR
      * locales:
-     *   en:
-     *     link: http://app_dev.php/en/... or http://app_dev.php?_locale=en
-     *     locale: en
```

Comment:
could i configure both a locale with region and without? would be nice to keep this example and just add en_US additionally, to not risk the assumption that you need the region even if you do not care about it.